tests/visualization_tests/test_timeline.py

```diff
@@ -0,0 +1,73 @@
+from io import BytesIO
+from typing import List
+
+import pytest
+
+import optuna
+from optuna.study.study import Study
+from optuna.trial import TrialState
+from optuna.visualization._plotly_imports import go
+from optuna.visualization._timeline import _get_timeline_info
+from optuna.visualization._timeline import plot_timeline
+
+
+@pytest.mark.parametrize(
+    "n",
+    [0, 3],
+)
+def test_get_timeline_info(n: int) -> None:
+    states = [TrialState.COMPLETE, TrialState.RUNNING, TrialState.WAITING]
+    study = optuna.create_study()
+    for i in range(n):
+        study.add_trial(
+            optuna.trial.create_trial(
+                params={"x": 0.5},
+                distributions={"x": optuna.distributions.FloatDistribution(0, 1)},
+                value=0,
+                state=states[i],
+            )
+        )
+    info = _get_timeline_info(study)
+    assert len(info.bars) == n
+    if n == 3:
+        trials = study.get_trials()
+        assert trials[0].datetime_start is not None
+        assert trials[1].datetime_start is not None
+        assert trials[2].datetime_start is None
+        assert trials[0].datetime_complete is not None
+        assert trials[1].datetime_complete is None
+        assert trials[2].datetime_complete is None
```

Comment:
These assertions are not for the test of timeline info. Could you remove them to keep the test simple?

Reply:
I see. I removed them.